AppKit/CPView.j

```diff
@@ -3511,7 +3703,7 @@ var CPViewAutoresizingMaskKey       = @"CPViewAutoresizingMask",
         [self _decodeThemeObjectsWithCoder:aCoder];
 
         [self setAppearance:[aCoder decodeObjectForKey:CPViewAppearanceKey]];
-
+        
```

Comment:
Sorry for this. "Automatically trim trailing whitespace" was checked but not the sub-option. Done now.